docs/src/documentation/README.md

```diff
@@ -0,0 +1,22 @@
+# Einführung
+
+Die ISI Anwendung wird in der Microservice Architektur umgesetzt und in Github als Multi-Repos verwaltet.
+
+Derzeit werden die folgenden Services in Github unter der MIT Lizenz Open Source zur Verfügung gestellt:
+
+## ISI Frontend
```

Comment:
Im Repo für das Frontend und Backend liegt jeweils die selbe Beschreibung.

Alternativ könnten wir die docs in ein eigenes isi-Repo packen. z.B. isi-docs

Reply:
ja, das passt.